### ISSUE TITLE
azure_rm_keyvaultsecret_info

### ISSUE THREAD

**Issue by ElanShudnow** [Azure/azure_preview_modules#308]
Hi, can I request that this module be introduced?  I see azure_rm_keyvaultkey_info was introduced but would be great if we could also retrieve the value for a secret, not just a key.

**Comment by Fred-sun** [comment]
@yungezz @zikalno New module requirements -azure_rm_keyvaultsecret_info, Please take a look when you're free! Thanks a lot!

**Comment by Gvazzana** [comment]
Yes please! This would be huge for the Azure community. Not having to manually duplicate key vault secrets in the Ansible Vault, would be much more secure, and less overhead.
I am creating all of my infrastructure in terraform, creating key vault secrets at provision time. To then leverage these key vault secrets in Ansible to configure the VMs would be outstanding! Thank  you

**Comment by imjoseangel** [comment]
Added to PR 316

https://github.com/Azure/azure_preview_modules/pull/316

Please review

**Comment by Fred-sun** [comment]
@imjoseangel Thank you very much for your interest in Ansible. This repo is no longer maintained in this repository and has been migrated to https://github.com/ansible-collections/azure
Please re-submit this Issue in the above repository and closed this. Thank you very much!

**Comment by imjoseangel** [comment]
@ElanShudnow @Fred-sun The PR for this issue has already been merged in the collections repository. Do not re-submit a new one as it is not needed. Thank you!
